src/backend/services/CommandInterpreter.ts

```
@@ -0,0 +1,50 @@
+        );
+
+      case 'REPLACE_LOCATION':
+        return this.stateManager.replaceLocation(
+          command.payload.scenarioId,
+          command.payload.oldLocation,
+          command.payload.newLocation
+        );
+
+      default:
+        // This ensures that if a new command is added to the type but not here, TypeScript will error.
+        const _exhaustiveCheck: never = command;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)